app/views/clients/viewclient.html

```diff
@@ -965,11 +965,13 @@ <h4><strong>{{client.displayName}}</strong></h4>
 							<td>{{clientdocument.fileName}}</td>
 							<td>
 								<a target="_blank" href="{{hostUrl}}{{clientdocument.docUrl}}"><i class="fa fa-cloud-download"></i></a>&nbsp;
+								<button class="btn btn-default" ng-click="previewDocument(clientdocument.docUrl,clientdocument.fileName)"> Preview </button>
 								<a ng-click="deleteDocument(clientdocument.id,$index)"><i class="fa fa-times"></i></a>
 							</td>
 						</tr>
 						</tbody>
 					</table>
+					<object height="100%" width="100%" ng-show="preview" data="{{fileUrl}}" type="{{fileType}}"></object>
```

Comment:
please make this a fixed width and height.

Reply:
Ok, what values should I give?